src/rohan/StringScan.java

```diff
@@ -0,0 +1,58 @@
+package rohan;
+
+import java.util.Scanner;
+
+/*Coding Exam 1: 8th Nov'2020
+1) print name having maximum length and print its length also.
+Add Adv : if you can scanner class.
+String[] name = {"Harsh","maulik","Technocredits","Krishna"};
+output : Technocredits -> 13
+
+String getMaxLengthName(String[] arr){
+{
+return name;
+}
+
+2) display method to print all even charcter of the name having maximum length.
+void display(String name){
+
+}*/
+
+public class StringScan {
+
+	String getMaxLength(String[] arr) {
+		String maxStr = arr[0];
+		for(int i=0;i<arr.length;i++) {
```

Comment:
we can reduce one comparison here as you have already stored arr[0] to maxStr. we can directly start loop from i =1.

Please use index instead of i.